### ISSUE TITLE
Not able to access "shares" in message

### ISSUE THREAD

**Issue by ajeetsk** [restfb/restfb#405]
Not able to access shared location on Message thread. I doubt the feature itself is not available with restfb. Is it going to be released soon? If it is there, can you pls help me with this?

Thanks,
Ajeet

**Comment by nbartels** [comment]
We have to add this element https://developers.facebook.com/docs/graph-api/reference/v2.5/message/shares/

**Comment by nbartels** [comment]
I added the change and it is released in version 1.21.0. 

You may use a custom solution until the next version is released.

For example this way:

```java
class CustomMessage extends Message {

    @Facebook
    private List<JsonObject> shares = new ArrayList<JsonObject>();

    public List<JsonObject> getJsonShares() {
      return shares;
    }
}
```

I used the `JsonObject`, but you may copy the `Share`-object from the commit above.

**Comment by ajeetsk** [comment]
Thanks for the prompt help.
